packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -145,4 +121,8 @@ export default class ScrubbingCenterController {
 
     return this.getAllEvents(value.ipBlock);
   }
+
+  showStats(row) {
```

Comment:
Use routing resolve to do this action. Doing so, you don't need to make $state accessible to the scope